book/website/afterword/legal-disclaimer.md

```diff
@@ -0,0 +1,46 @@
+# Legal disclaimer: The Turing Way is not legal advice!
+
+## Prerequisites
+| Prerequisite | Importance | Notes |
+| -------------|----------|------|
+| None | n/a | n/a |
+
+## Summary
+The Turing Way is a lightly opinionated guide to reproducible research. It is *not* a source of legal advice and should never be treated as such. 
+
+## Good legal advice is timely, specific, and given by a legal expert 
+The Turing Way is not these things. 
+
+The Turing Way is not written to be a source of legal expertise. 
+When laws and regulations are referenced, the information provided is heavily simplified. 
+The intent is to give you an overview of the basics, so that you will know when to check whether something you want to do has potential legal ramifications. 
+
+Don’t make any important decisions based solely on information about laws and regulations in this book..
+
+### Finding legal expertise
+If you are part of an institution (for example, a university) or an organisation (for example, a non-profit), it is very likely that you can access legal advice either in-house or from a specialist firm that already works with your institution or organisation. 
```

Comment:
This might be a bit a small thing, but is the institution/organisation distinction important? I am also unsure the "non-profit" example clarifies much, as some might take universities to be non-profits. Could this simply say "If you are part of an organisation such as a university, a charity or a for-profit company, [...]"?

Reply:
```suggestion
If you are part of an organisation such as a university, a non-profit or a company, it is very likely that you can access legal advice either in-house or from a specialist firm that already works with your institution or organisation. 
```